### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by lucymk**
- gets video URL from video object on technique page
- sets page min height to innerHeight of window to account for mobile menus

### CODE REVIEW

File: components/Page.js [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-  className: 'min-h-content',
-})``
+const PageStyled = styled.main.attrs({})``
 
 const Page = ({ _type, title, children }) => {
+  const [windowHeight, setWindowHeight] = useState('100vh')
   const navbarOptions = getNavbarOptions({ _type, title })
 
+  useEffect(() => {
+    // footer = 200ox, top navbar = 60x
```
Comment (maxgerber) [inline components/Page.js:16]: Typos?

File: components/Onboarding.js [hunk 2]
```diff
@@ -17,7 +17,11 @@ const Onboarding = () => {
   const [onboarding, setOnboarding] = useLocalStorage('onboarding', null)
   const closeOnboarding = () => setOnboarding(false)
 
-  useEffect(() => onboarding === null && setOnboarding(true), [])
+  useEffect(() => {
+    if (onboarding === null) {
```
Comment (maxgerber) [inline components/Onboarding.js:21]: Was this not werqin?

Comment (lucymk) [reply components/Onboarding.js:21]: was giving me linting errorrrrrsss 😫

File: components/Page.js (hunk 1 above)
Comment (lucymk) [reply components/Page.js:16]: haha 200 🐂

**Pull request merged**